test/calculator_test[1]_include.cmake

```diff
@@ -0,0 +1,5 @@
+if(EXISTS "/mnt/c/Users/jared.vandewater/Learning/CalculatorSkeleton/test/calculator_test[1]_tests.cmake")
```

Comment:
this file should not be part of the project... remove it

Reply:
I'm not sure what this is or how it was created.. weird